sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/models/DocumentFieldType.java

```diff
@@ -11,6 +11,13 @@
 /** Defines values for DocumentFieldType. */
 @Immutable
 public final class DocumentFieldType extends ExpandableStringEnum<DocumentFieldType> {
+
+    /**
+     * Constructs a DocumentFieldType object.
+     */
+    public DocumentFieldType() {
```

Comment:
Since this is an `ExpandableStringEnum` this will need to be marked as deprecated and tell users to use the factory method.